src/tests/poc/new-workflow/scaling-test.cpp

```diff
@@ -59,8 +67,7 @@ BOOST_DATA_TEST_CASE(test_scaling_simple_problem, bdata::make(solverNames)*bdata
     // Create thermal production units
     // Every unit can produce up to 10 MW
     // Price is stable in time but increase for every unit
-    int nUnits = ceil(1.0 * (nTimesteps - 1) / 10.0);
-    for (int i = 1; i <= nUnits; ++i) {
+    for (int i = 1; i <= nbUnits; ++i) {
         string id = "thermal" + to_string(i);
         Component thermal(id, THERMAL, {{"maxP", 10}}, {});
```

Comment:
The MIP may be infeasible since consumption increases up to "nTimesteps - 1" MW (this test now fails)
Increase Pmax to avoid infeasible problems